### DISCUSSION TITLE
how to fix all assets deleted after first request to server

### DISCUSSION THREAD

**Discussion by putu-eka-mulyana** [adonisjs/core#2986] [Help]
how to fix all assets deleted after first request to server ? i use adonisjs version 5.1.0 and node js version 14.17.0

![image](https://user-images.githubusercontent.com/28705956/127763028-e8bbb2e1-4c7a-478f-8fb5-8d8148fe96bd.png)

**Comment by thetutlage** [comment]
Why you think this is a problem?

**Reply by putu-eka-mulyana** [reply]
because all my assets are deleted

**Reply by thetutlage** [reply]
Those are the compiled assets and not the source. The source should be in the resources directory

**Comment by McSneaky** [comment]
> how to fix all assets deleted after first request to server ? i use adonisjs version 5.1.0 and node js version 14.17.0

Do all your assets go missing and page breaks, or it's just deleting old ones?
Also is this happening with every request or just once when you start up server?
